tests/translator/output/aws-cn/connector_api_to_multiple_function.json

```diff
@@ -7,6 +7,8 @@
             "Destination": {
               "Type": "AWS::Lambda::Function"
             },
+            "Embedded": false,
+            "MultiDestination": true,
```

Comment:
This was originally a single connector, right? In the output now we have to `Metadata`s with `MultiDestination` being true. Is that the intention? How do we know how many times multi-destination was used?

Reply:
That's actually a great point. I was thinking that for multi-destination connector, each generated policy should count as one usage. But I should double check with Simon on this behaviour.